### PR TITLE
Remove tasks for analyzing the repo graph

### PR DESCRIPTION
**Pull request by natemcmaster**
As a part of converging repos, we no longer need this code. It is currently buggy and slow, and we can get away without by hard-coding the repo graph (which rarely changes)

Resolves https://github.com/aspnet/AspNetCore-Internal/issues/1135

### CODE REVIEW

File: build/repo.targets [hunk 1]
```diff
@@ -226,12 +223,6 @@
   </Target>
 
   <Target Name="ComputeGraph" DependsOnTargets="ResolveRepoInfo;GeneratePropsFiles">
```
Comment (ryanbrandenburg) [inline build/repo.targets]: Technically this is more like "LoadSubmoduleGraph" now but it's probably not worth changing the name.

**Comment by ryanbrandenburg** [comment]
How does it resolve that internal issue though? Did I miss one of the failures?

**Comment by natemcmaster** [comment]
The error "The SDK 'Internal.AspNetCore.Sdk' specified could not be found." was being thrown from the ComputeRepoGraph task. Something about MSBuild's programmatic APIs is non-deterministic when resolving custom SDKs.

**Comment by ryanbrandenburg** [comment]
Spooky. :shipit:

**Pull request merged**